Src/utils.cpp

```diff
@@ -0,0 +1,5 @@
+#include "utils.hpp"
+
+namespace common_utils {
```

Comment:
first namespace should be project namespace.
```suggestion
namespace template_project::common_utils {
```